<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormTypes</class>
 <widget class="QDialog" name="FormTypes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>891</width>
    <height>567</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>787</width>
    <height>523</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Типы узлов и рёбер</string>
  </property>
  <widget class="QTableView" name="tvNodeTypes">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>221</width>
     <height>471</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tvEdgeTypes">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>421</width>
     <height>471</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_dst_private">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>490</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_n_types">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>0</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Типы узлов</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_e_types">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>0</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Типы рёбер</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="push_n_add">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string> Добав.</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="push_n_save">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Сохр.</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="push_n_revert">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Отм.</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="push_n_del">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Удал.</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>530</y>
     <width>401</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="push_e_add">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string> Добавить</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="push_e_save">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Сохранить</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="push_e_revert">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Отменить</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="push_e_del">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>500</y>
     <width>401</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QSpinBox" name="spin_e_id">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cb_src">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="line_edge_name">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cb_dst">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="cb_src_private">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>490</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_e_types_2">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>0</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Типы кванторов</string>
   </property>
  </widget>
  <widget class="QTableView" name="tvQuantTypes">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>20</y>
     <width>251</width>
     <height>471</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QSpinBox" name="spin_n_id">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="line_node_name">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>500</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QSpinBox" name="spin_q_id">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="line_quant_name">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>530</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QPushButton" name="push_q_add">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string> Добав.</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="push_q_save">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Сохр.</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="push_q_revert">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Отм.</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="push_q_del">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Удал.</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>tvNodeTypes</zorder>
  <zorder>tvEdgeTypes</zorder>
  <zorder>label_n_types</zorder>
  <zorder>label_e_types</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>cb_dst_private</zorder>
  <zorder>cb_src_private</zorder>
  <zorder>label_e_types_2</zorder>
  <zorder>tvQuantTypes</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
